README.md

```diff
@@ -43,13 +38,16 @@ From the rootdir, paste the command that printed to your terminal. This command
 cat server/dumps/head.sql server/dumps/copySpaces.sql server/dumps/spacesBody0.sql server/dumps/spacesBody1.sql server/dumps/copyReviews.sql server/dumps/reviewsBody0.sql server/dumps/reviewsBody1.sql server/dumps/copyCalculations.sql server/dumps/calculationsBody0.sql server/dumps/calculationsBody1.sql server/dumps/foot.sql > server/dumps/dump.sql
 ```
 
+4. Once the "dump.sql" file is created, from the root directory of the project in your terminal, run the command "docker-compose --build up pg". This will read the "pg" service in the "docker-compose.yaml" file and only build that image and run a container, while ignoring the other services. Once the image is build, the Postgres container will see the "dump.sql" file and begin copying the information to the database. This process should take very little time if you set the number of iterations to 0 or 1 earlier. You will notice the creation of another folder called "pg" in the root directory of the project, which is just a volume to persist the data that was seeded so that you do not need to seed the database when stopping and restarting a Postgres container.
```

Comment:
Is the dump.sql file the outcome of the concatenated files from before?

Reply:
Yes, that's right